src/settings.cpp

```diff
@@ -548,6 +548,16 @@ static void Write_ValidateStdString(void *ptr, const SettingDesc *sd, const char
 	}
 }
 
+/**
+ * Read the string from the the actual setting.
+ * @param object The object the setting is to be saved in.
+ * @return The value of the saved string.
+ */
+const std::string &StringSettingDesc::Read(const void *object) const
```

Comment:
Is this function in preparation of future work? (which I can fully understand). It feels out of place here atm, hence the question :)

Reply:
Yeah, I saw. But what I see in this commit is a function with a single line of code called exactly once. That feels a bit out of place. Hence the question if this is used more in the future, or if it is intended to be like this?

Reply:
Yes, here in `FormatValue`. Later in this PR in `IsSameValue`. Though it is not used as often as the same function for `IntSettingDesc`.

Reply:
Okay, but that is fine. It just looks odd in this one commit, but who cares :) As long as it is useful :D